Source/Web View/WebViewController.swift

```diff
@@ -778,6 +778,7 @@ public extension NSObject {
         }
     }
     
+    @objc
```

Comment:
`@objc` was added to the same line as `func` definition for other functions in the file. 
Can we keep it consistent?

Reply:
Yeah the only reason I added it on the line above was because the computed property directly above it uses that style: https://github.com/Electrode-iOS/ELHybridWeb/blob/f27a4008982b4cdcc65e26cca0ffbe8e1fdbadd3/Source/Web%20View/WebViewController.swift#L770-L771

I'll update both to be the same line for consistency with the rest of the file.